### ISSUE TITLE
KCD Los Angeles 2024 (March)

### ISSUE THREAD

**Issue by jberkus** [cncf/kubernetes-community-days#510]
# KCD Organizer Checklist
Please fill in information, organizers, and sponsors as you assemble them, and then make a new comment when this application is ready for review.

### Event Details
* Event Location: Pasadena, CA, USA
* Event Date and Time: March 14-15, 2024
* Expected Number of Attendees: 200 to 250
* Event cost per ticket, and if you will have early bird pricing or not: Included in SCALE general pass for $80

#### Event type:
* [ ] Virtual
* [ ] Hybrid
* [X] In person

#### Organizers Details (identify at least 3 organizers). Designate who will be the CFP admin, as well as the finanical officer:
* [x] Organizer 1: Josh Berkus, Red Hat, Kubernetes Community Manager, @jberkus / CFP Admin & Financial (for now)
* [x] Organizer 2: Steve Wong, VMware, Software Engineer, @steve-wong
* [x] Organizer 3: Justin Garrison, AWS, DevRel, @rothgar
* [x] Organizer 4: Guinevere Saenger, Software Engineer, Pulumi, @gsaenger
* [x] Organizer 5: Faisal Azfal, nClouds, Solutions Architect, @Faisal
* [x] Organizer 6: Savitha Raghunathan, Software Engineer, Red Hat, @sraghunathan

### Terms

* [X] No other Kubernetes Community Day event in same major city (check with past organizers)
* [X] Minimum of 3 organizers from 3 different organizations
* [x] Every organizer has an account on [Cloud Native Community Groups](https://community.cncf.io/)
* [X] Each organizer lives within the KCD city or just outside of it. For small countries, the organizers can live within the region, but must be able to easily travel to the KCD if it were in-person.
* [X] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer. NOTE: An employee of a member company, qualifies.
* [X] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [X] All organizer inquiries / communication for CNCF must be summarized via GitHub (comment under your KCD issue) for easy tracking
* [ ] All organizers have taken [Inclusive Open Source Community Orientation (LFC102)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* [ ] Agree to offer at least 3 complimentary tickets to increase diversity for the event
* [x] Share the sponsor prospectus with CNCF via your GitHub issue, and link it on your website.
* [X] Agree to ensure the final program schedule is diverse (e.g., not all speakers of one gender, one culture, or from 1 or 2 of the same company). [Here are some tips on how to create a diverse lineup.](https://docs.google.com/presentation/d/1fzT_BdavVKh3mnxxU-PBWyJq9JUfasKwHqekkbYVbw8/edit#slide=id.g56245ab439_0_106)
* [X] Agree to submit your final lineup to kcd@cncf.io for review, prior to publishing it
* [ ] Agree to provide CNCF with a full list of attendees for the KCD from the SCALE registration platform.
-- Regardless of your platform choice, please use [these questions](https://docs.google.com/spreadsheets/d/1OEoVQ8Y3eQmEEJeT2abhNOkYIkyk8nAVjebNFvKMTQQ/edit?usp=sharing) to create your registration form

As with last year, KCD-LA will be using SoCalLinuxExpo's platform.  As such, the above two points do not apply.  We need to have a discussion around an event survey to make up some of the missing data from the registration.

### Getting started checklist for organizers

* [x] Schedule monthly check-ins with KCD Core Organizer(s) (kcd@cncf.io)
* [ ] Minimum of 3 sponsors committed (not required for virtual only events)
Sponsor 1
Sponsor 2
Sponsor 3

We do not have sponsors yet.  However, the event is already scheduled and will take place regardless of fundraising.

* [X] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
Fiscal sponsor that will accept payments
* [X] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel
* [ ] Create a ~Twitter~ Mastodon handle specific to your KCD event (except for countries like China and Korea where Twitter is not used)

### Getting started checklist for CNCF:

* [ ] Organizer email alias has been created
* [ ] Region created in Bevy
* [ ] Added to Bevy header
* [ ] CFP created in SMapply
* [ ] Marketing Plan shared

### Organizer Planning Timeline
* [ ] Make a copy of [this timeline](https://docs.google.com/spreadsheets/d/1bvCiyyDut1seSnBE6pzVevcJkXLeWbxbncvhFsyY8PI/edit#gid=1736864170) based on whether your event is In-person, Hybrid, or Virtual and link your own spreadsheet here ().
* [ ] To assure you have a diverse lineup, you are required to send a copy of your program to kcd@cncf.io before you publish it.

### Post event checklist for Organizers:
* [ ] Send CNCF event analytics
* [ ] Schedule post mortem meeting
* [ ] Send CNCF videos to be uploaded to YouTube
* [ ] Link videos from YouTube to event page
* [ ] Post event survey sent out
* [ ] Fill out [Credly badge request form](https://docs.google.com/forms/d/e/1FAIpQLSex2EhkCNysmK7svQEVeaOlqpsH0fSBe6qoOePRw_WGajZnTw/viewform) for 1 badge per KCD

### Post event checklist for CNCF:
* [ ] Removed from Bevy header
* [ ] CFP deactivated
* [ ] Upload videos to YouTube

**Comment by AudMonte01** [comment]
AUGUST 17 MEETING UPDATE

Content
* 1 Keynote speaker. Potentially (Marino Wijay)[https://www.linkedin.com/in/mwijay/]
* CFP submissions so far 26 in review and 9 in pending. Mostly international submissions
* Looking to double submissions in the next two weeks 
* Chad is directly contacting people
* CNCF has CFP in KubeWeekly and socializing

Sponsors
* IBM is interested in sponsoring
* DELL will not sponsor
* Google open source will not sponsor monetarily (will check to see if they will be 
* K8s Learning will be partnering to help promote

Marketing
* Bart Farrell offered to do a promo video for the agenda with speakers (early September)
* [ ] Chad to send a blog post to get published on CNCF blog by the end this week

**Comment by AudMonte01** [comment]
AUGUST 17 KICKOFF MEETING NOTES

Platforms
* Registration for the evening social on KCD Bevy page

Organizers
* New organizer: Faisal Afzal is new to the committee and works with ArgoCD

Content
* SCALE CFP opens closes November 1
* Workshops on Thursday?
* Percona proposed DoKC co-located content as a featured community
* Justin is attending a meetup in Irvine next week and is going to promote the CFP
* Justin is also speaking at DockerCon in Oct in LA which he can promote there as well

Sponsors
* Money goes through SCALE
* Keeping it similar to DevOps Days prospectus
* Offering will be logo placement and tables set outside the room

Ancillary event
* Social event for this KCD specific
* Invite folks to K8s LA community (Justin runs the meetup)
* Co-working space around the corner from SCALE for 150 people max (not WeWork)
* OR Doghouse buyout (order over counter)

Actions
* [ ] Josh to share to CFP link with Audra to promote in KubeWeekly newsletter and socials

**Comment by faisal** [comment]
```
$ node
Welcome to Node.js v20.5.1.
Type ".help" for more information.
> "Azfal" === "Jawdat"
false
```

Do you mean @faisalcodesinfrastructure ?

**Comment by AudMonte01** [comment]
SEPTEMBER 12 MEETING NOTES

Content
* Handful of submissions thus far
* SCALE is hosting an online CFP best practices session

Social channels
* Mastadon and LinkedIn
* Don't forget to use [#socialmedia](https://cloud-native.slack.com/archives/C12MRQ97A) to post 
* Audra to add to KubeWeekly (missed previous timing)

Action
* [ ] Josh to send Audra PDF of sponsor prospectus to add to KCD prospectus repo

**Comment by AudMonte01** [comment]
OCTOBER 10 MEETING NOTES

Sponsorships
* Are hurting due to KubeCon Paris, so might not be able to have a social or sponsor speaker travel

Content
* 22 submissions and 3 weeks out (not including DoKC)
* 26 potential speakers showed up to CFP writing workshop last Friday

Marketing
* Use KCD booth at KubeCon Chicago for curated content
 * [ ] Team to decided how they want to use 10 LF exam vouchers. Audra to send codes.

**Comment by AudMonte01** [comment]
NOVEMBER 14 MEETING NOTES

Content
* Talks are under review. Justin created an app for reviewing
* KubeCon speaker candidates were planning to go to Paris
* Potential for offering Credly badges to speakers
* First sponsor efforts to get a sponsor for speaker travel

Other marketing efforts
* [ ] Audra to send sticker voucher code to team and let them know once CNCF store is replenished

**Comment by AudMonte01** [comment]
MEETING NOTES

Content
* Notifications started going out last Thursday. Waiting on confirmation from speakers
* Agenda due to go LIVE in January
* Shout-out to sponsors is offered, but not talks (2-minutes)
* 7% of submissions were stealth product pitches (present proprietary stuff)
* The list of speakers is firm, but the order may have some shuffle
* DoK will be a virtual track (not online), but sharing a room across all tracks
* 90% of the audience

Sponsors
* Isovalent is confirmed
* Media sponsor is Learned K8s
* Other sponsors can be speaker T&L recognition, for example

Logo
* [ ] Redesign with elements of year 1 header image into the new logo's creative space. Josh to send Audra high-resolution image to use

**Comment by AudMonte01** [comment]
JANUARY 9 MEETING NOTES

Schedule LIVE
* Waiting for confirmation from two speakers

CNCF marketing
* [ ] Audra to inquire about CNCF booth on-site at SCALE, and staffing

**Comment by AudMonte01** [comment]
FEBRUARY 6 CNCF SYNC

Sponsors
* More sponsors came through at the Gold level
* Audra helped re-work the Bevy Page to have a better flow
* [ ] Audra to see if Taylor Dorzal, Head of CNCF Ecosystem could possibly attend this KCD at least on Friday

Content
* Schedule is LIVE on both Bevy and SCALE 21x sites
* [x] Audra to plug schedule in KubeWeekly newsletter

Other Actions
* [ ] Audra to get training vouchers to the team

**Comment by AudMonte01** [comment]
APRIL 4 RETROSPECTIVE SYNC

Analytics
* Total checked in = 101+ unique people
* Total speakers with non-male speaker percentage = 2 out of 12 KCD session (16%)
* Total sponsors with insight into profit or loss = 4-5 KCD-specific sponsors that covered F&B for the KCD. Operated at a bit of a loss with SCALE due to KubeCon Paris (and St. Patrick's Day?) being the same weekend

[Pros and Cons of the event can be found here](https://docs.google.com/document/d/1mtDOl9UrbMXt3XRjlFUbYEnjM-3P7zPN6NtS3r8GGeg/edit)

Post-event marketing
* [ ] Audra to share transparency report template with Justin and share in CNCF Slack channel
* [ ] Videos already uploaded to YouTube - Audra needs to duplicate 
* [ ] Justin to write Blog post on cncf.io
